modules/tinymce/src/core/test/ts/browser/selection/SelectionBookmarkInlineEditorTest.ts

```diff
@@ -60,16 +60,21 @@ describe('browser.tinymce.core.selection.SelectionBookmarkInlineEditorTest', ()
   before(() => addTestDiv());
   after(() => removeTestDiv());
 
+  it('assert bookmark is updated in response to `setRng`', () => {
+    const editor = hook.editor();
+    editor.resetContent('<p>a</p><p>b</p>');
+
+    TinySelections.setRawSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 0);
+    TinySelections.setSelection(editor, [ 1, 0 ], 1, [ 1, 0 ], 1, false); // Ensure node change doesn't fire
```

Comment:
Probably just missing context, but  we need to set raw selection then selection here? And it doesn't matter that the following case sets selection then raw selection - so in the opposite order?

Reply:
This one is verifying that `setSelection()` changes the bookmark.

The following case is verifying that moving the cursor without a `nodechange` event doesn't update the bookmark - so we can't use `setSelection()`.

In theory more of these tests should use raw selection now but I didn't want to get too far off track.